src/update_agent/actor.rs

```diff
@@ -190,12 +190,16 @@ impl UpdateAgent {
             return Box::new(actix::fut::err(()));
         }
 
+        log::info!(
+            "new deployment '{}' selected, proceeding to stage it",
```

Comment:
Super minor: so, "deployment" has a very particular meaning in rpm-ostree. It more or less means "checked out OSTree commit". But we haven't deployed anything yet. Maybe e.g. `release` or `version` would fit better here?